<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePathDlg</class>
 <widget class="QDialog" name="CreatePathDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>生成轨迹</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>生成参数</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
        <property name="leftMargin">
         <number>30</number>
        </property>
        <property name="verticalSpacing">
         <number>12</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>加工方法</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>切削面</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QComboBox" name="cmbToolNumber"/>
        </item>
        <item row="8" column="1">
         <widget class="QPlainTextEdit" name="editAfterCode">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>mm</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>选择刀具</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPlainTextEdit" name="editBeforeCode">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>路径执行后附加G代码</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>加工精度</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QDoubleSpinBox" name="editUTolerance">
          <property name="maximum">
           <double>100000.000000000000000</double>
          </property>
          <property name="value">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>路径执行前附加G代码</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbOpType">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>面上沿边界往返</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>沿边界单向</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QListWidget" name="listFaces"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>轨迹线</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QListWidget" name="listCurves"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbToolNumber</tabstop>
  <tabstop>editBeforeCode</tabstop>
  <tabstop>editAfterCode</tabstop>
  <tabstop>editUTolerance</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
